package.json

```diff
@@ -1,7 +1,6 @@
 {
-  "name": "mongodb-compass",
-  "productName": "MongoDB Compass",
-  "description": "The MongoDB GUI",
+  "name": "mongodb-compass-monorepo",
```

Comment:
Seems to be [a bug in lerna](https://github.com/lerna/lerna/issues/2064) that doesn't run root scripts if the root package name matches one of the packages, hence the update